commit/core.pyx

```diff
@@ -657,6 +657,15 @@ cdef class Evaluation :
         config.nIC      = self.KERNELS['wmr'].shape[0]
         config.nEC      = self.KERNELS['wmh'].shape[0]
         config.nISO     = self.KERNELS['iso'].shape[0]
+        config.locally  = build_locally
+
+        if build_locally:
```

Comment:
Would it be better to only have a `pyximport.install` with variables depending on the `build_locally`?